### ISSUE TITLE
Train AFHQ V2 dataset

### ISSUE THREAD

**Issue by 49xxy** [NVlabs/stylegan3#189]
Thank you very much for your excellent work!
At present, I want to use my own dataset for training, I refer to your command for training with AFHQ V2 dataset. 
`python train.py --outdir=~/training-runs --cfg=stylegan3-t --data=~/datasets/afhqv2-512x512.zip \
    --gpus=8 --batch=32 --gamma=8.2 --mirror=1
`
Although there are three classes of AFHQ V2 dataset, but you are trained unconditionally, so I want to know what the "dataset. Json " file in this dataset is like, is it unlabeled?As shown in the following：
![image](https://user-images.githubusercontent.com/102204219/182054919-3ba2fae0-0780-42e0-9dcd-5d02d4181c9e.png)

**Comment by PDillis** [comment]
Yes, if there are no labels, then you should only get inside the `dataset.json` a `{"labels": null}` and your model will be an unconditional one. This means you don't need to pass a class to it when generating new images/videos, but all 'classes' will be mixed, if there are any within the dataset.
